hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -677,6 +677,7 @@ message KeyArgs {
     // request type.
     optional uint64 modificationTime = 13;
     optional bool sortDatanodes = 14;
+    optional bool refreshPipeline = 15;
```

Comment:
Do we need this flag at all? As from the code it looks like we always want to return with the latest block information or are there any case we want to send it to false?. So, in OzoneManagerRequestHandler, we can just set omKeyArgs.set(true) right?

Similar is done for other requests like lookupFile.

Reply:
Yes, I agree. I can make the change.